src/frappe/userstyle.css

```diff
@@ -1,38 +1,12 @@
 /* Style for rendered Markdown */
 :root {
-  --ctp-frappe-rosewater: #f2d5cf;
-	--ctp-frappe-flamingo: #eebebe;
-	--ctp-frappe-pink: #f4b8e4;
-	--ctp-frappe-mauve: #ca9ee6;
-	--ctp-frappe-red: #e78284;
-	--ctp-frappe-maroon: #ea999c;
-	--ctp-frappe-peach: #ef9f76;
-	--ctp-frappe-yellow: #e5c890;
-	--ctp-frappe-green: #a6d189;
-	--ctp-frappe-teal: #81c8be;
-	--ctp-frappe-sky: #99d1db;
-	--ctp-frappe-sapphire: #85c1dc;
-	--ctp-frappe-blue: #8caaee;
-	--ctp-frappe-lavender: #babbf1;
-	--ctp-frappe-text: #c6ceef;
-	--ctp-frappe-subtext1: #b5bddc;
-	--ctp-frappe-subtext0: #a5acc9;
-	--ctp-frappe-overlay2: #949bb7;
-	--ctp-frappe-overlay1: #838aa4;
-	--ctp-frappe-overlay0: #737891;
-	--ctp-frappe-surface2: #62677e;
-	--ctp-frappe-surface1: #51566c;
-	--ctp-frappe-surface0: #414559;
-	--ctp-frappe-base: #303446;
-	--ctp-frappe-mantle: #292c3c;
-	--ctp-frappe-crust: #232634;
```

Comment:
I've opted to go with removing the variables entirely and using Whiskers variables below instead as it accomplishes the same thing in a less verbose way (though it does make the diff more complicated).